modyn/supervisor/internal/triggers/drift/embedding/embeddings.py

```diff
@@ -0,0 +1,40 @@
+import torch
+from torch.utils.data import DataLoader
+
+from modyn.supervisor.internal.triggers.utils.model.manager import ModelManager
+
+
+def get_embeddings(model_manager: ModelManager, dataloader: DataLoader) -> torch.Tensor:
```

Comment:
This needs a test

Reply:
Sure, I'll add one. This function also didn't have one before, though😅